src/test/java/alison/ParserTest.java

```diff
@@ -0,0 +1,18 @@
+package alison;
+
+import alison.exception.AlisonException;
+import alison.task.Deadline;
+import alison.tool.Parser;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Perhaps a line break between lines 5 and 6 and lines 6 and 7 would make the import statements look neater? 😄